v2/src/components/appInfoForm/style.css

```diff
@@ -52,4 +32,98 @@
 .button:hover {
   background-color: #ff8d1c;
   color: #ffffff;
+}
+
+.app-info-form-label {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
+
+/* Explanation popup styles */
+.app-info-form-explanation {
+  position: fixed;
+  transform: translateX(-50%) translateY(-10px);
```

Comment:
On Safari on iOS the box is not directly under the question mark icon, and sometimes it renders of screen. Translation and skew sometimes doesnt work directly and the webkit counterparts need to be used so double check that 

(Cant confirm on desktop because the preview deployment was cancelled and my current branch for docs is in a state I cant revert)